utils/cmake/dreamcast.cmake

```diff
@@ -0,0 +1,2 @@
+message(AUTHOR_WARNING "dreamcast.cmake is deprecated, please use kallistios.cmake")
+include($ENV{KOS_BASE}/utils/cmake/kallistios.cmake)
```

Comment:
This script is sourced manually, so after the toolchain script is loaded - as a result `include(kallistios)` should actually work.